lib/puppet/pops/lookup/lookup_adapter.rb

```diff
@@ -368,7 +368,7 @@ def initialize_module_provider(lookup_invocation, module_name)
       elsif mp_config.version >= 5
         unless provider_name.nil? || Puppet[:strict] == :off
           Puppet.warn_once(:deprecation, "metadata.json#data_provider-#{module_name}",
-            "Defining \"data_provider\": \"#{provider_name}\" in metadata.json is deprecated. It is ignored since a '#{HieraConfig::CONFIG_FILE_NAME}' with version >= 5 is present", mod.metadata_file)
+            _("Defining \"data_provider\": \"%{name}\" in metadata.json is deprecated. It is ignored since a '%{config}' with version >= 5 is present") % { name: provider_name, config: HieraConfig::CONFIG_FILE_NAME }, mod.metadata_file)
```

Comment:
@Magisus metadata.json refers to a filename, should it be left out of translation?